main/src/test/java/org/mobilitydata/gtfsvalidator/validator/FeedServiceDateValidatorTest.java

```diff
@@ -0,0 +1,133 @@
+package org.mobilitydata.gtfsvalidator.validator;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import java.util.Locale;
+import org.junit.Test;
+import org.mobilitydata.gtfsvalidator.notice.NoticeContainer;
+import org.mobilitydata.gtfsvalidator.notice.StartAndEndDateOutOfOrderNotice;
+import org.mobilitydata.gtfsvalidator.table.GtfsFeedInfo;
+import org.mobilitydata.gtfsvalidator.table.GtfsFeedInfoTableContainer;
+import org.mobilitydata.gtfsvalidator.type.GtfsDate;
+
+public class FeedServiceDateValidatorTest {
+
+  private static GtfsFeedInfoTableContainer createFeedInfoTable(
+      NoticeContainer noticeContainer, List<GtfsFeedInfo> entities) {
+    return GtfsFeedInfoTableContainer.forEntities(entities, noticeContainer);
+  }
+
+  public static GtfsFeedInfo createFeedInfo(
+      long csvRowNumber,
+      String feedPublisherName,
+      String feedPublisherUrl,
+      Locale feedLang,
+      GtfsDate feedStartDate,
+      GtfsDate feedEndDate) {
+    return new GtfsFeedInfo.Builder()
+        .setCsvRowNumber(csvRowNumber)
+        .setFeedPublisherName(feedPublisherName)
+        .setFeedPublisherUrl(feedPublisherUrl)
+        .setFeedLang(feedLang)
+        .setDefaultLang(null)
+        .setFeedStartDate(feedStartDate)
+        .setFeedEndDate(feedEndDate)
+        .setFeedVersion(null)
+        .setFeedContactEmail(null)
+        .setFeedContactUrl(null)
+        .build();
+  }
+
+  @Test
+  public void startDateBeforeEndDateShouldNotGenerateNotice() {
+    NoticeContainer noticeContainer = new NoticeContainer();
+    GtfsFeedInfoTableContainer gtfsFeedInfoTable =
+        createFeedInfoTable(
+            noticeContainer,
+            ImmutableList.of(
+                createFeedInfo(
+                    1,
+                    "name value",
+                    "url value",
+                    Locale.CANADA,
+                    GtfsDate.fromEpochDay(340),
+                    GtfsDate.fromEpochDay(450))));
+
+    FeedServiceDateValidator underTest = new FeedServiceDateValidator();
+    underTest.feedInfoTable = gtfsFeedInfoTable;
+
+    underTest.validate(noticeContainer);
+    assertThat(noticeContainer.getValidationNotices().isEmpty());
```

Comment:
This test and many others in that file do not test anything because of the wrong parenthesis.

That must be:
```
assertThat(noticeContainer.getValidationNotices()).isEmpty();
```

But it was:
```
assertThat(noticeContainer.getValidationNotices().isEmpty());
```

Reply:
Thanks for fixing this #815 https://github.com/MobilityData/gtfs-validator/pull/815#issuecomment-800760402